app/presenters/contents_list.rb

```diff
@@ -20,7 +20,8 @@ def contents_link(text, id)
         track_category: 'contentsClicked',
         track_action: 'leftColumnH2',
         track_label: id
-      })
+      }
+)
```

Comment:
I agree with @fofr - most of the changes in this PR are unrelated, lets see if we can drop them and still keep the linter happy.